src/common.h

```diff
@@ -90,6 +90,9 @@
 #define PRIaddress "u"
 #define PRIoffset PRIzx
 
+// Simd.
+typedef __m128 v128;
```

Comment:
I'm not sure what's "define this internally" means.  Move "typedef __m128 v128" to other header file?

BTW:
How should we deal with the platform that doesn't support 128-bit integer?

Reply:
Sorry, I meant define something in wabt directly, like:
```
struct v128 {
  uint32_t v[0];
};
```

We don't really need to use the native type, since most of the time we'll just be loading and storing the values anyway. The interpreter may want to use a real v128 type, but even there it might be nice to do it with scalar values instead.